master/pkg/schemas/copy_test.go

```diff
@@ -0,0 +1,57 @@
+package schemas
+
+import (
+	"bytes"
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+// Copy is the non-reflect version of copy, but mostly the reflect version is called from other
+// reflect code, so it's defined here in test code.
+func Copy(src interface{}) interface{} {
+	return cpy(reflect.ValueOf(src)).Interface()
+}
+
+func TestCopy(t *testing.T) {
```

Comment:
This doesn't seem to test anything

Reply:
oops, this is a weird vestige of a previous version.